Content/Items/Starwood/Weapons.StarwoodScepter.cs

```diff
@@ -0,0 +1,741 @@
+{
+	public class StarwoodScepterPool : LootPool
+	{
+		public override void AddLoot()
+		{
+			AddItem(ModContent.ItemType<StarwoodScepter>(), ChestRegionFlags.Surface, 0.5f, 1, true, 2);
+		}
+	}
+	public class DormantScepter : QuickMaterial
+	{
+		public DormantScepter() : base("Dormant Scepter", "The scepter lies dormant, waning for mystical energy", 1, Item.sellPrice(silver: 2), ItemRarityID.Gray, AssetDirectory.StarwoodItem + "StarwoodScepterDormant", true) { }
+	}
+
+	public class StarwoodScepter : StarwoodItem
+	{
+		public override string Texture => AssetDirectory.StarwoodItem + Name;
+
+		public StarwoodScepter() : base(ModContent.Request<Texture2D>(AssetDirectory.StarwoodItem + "StarwoodScepter_Alt").Value) { }
+
+		public override void SetStaticDefaults()
+		{
+			DisplayName.SetDefault("Starwood Scepter");
+			Tooltip.SetDefault("Summons two halves of a sentient star\nThe stars drop mana stars when hitting summon tagged enemies");
+
+			ItemID.Sets.GamepadWholeScreenUseRange[Item.type] = true; // This lets the Player target anywhere on the whole screen while using a controller.
+			ItemID.Sets.LockOnIgnoresCollision[Item.type] = true;
+		}
+
+		public override void SetDefaults()
+		{
+			Item.damage = 6;
+			Item.knockBack = 3f;
+			Item.mana = 10;
+			Item.width = 32;
+			Item.height = 32;
+			Item.useTime = 36;
+			Item.useAnimation = 36;
+			Item.useStyle = ItemUseStyleID.Swing;
+			Item.value = Item.buyPrice(silver: 25);
+			Item.rare = ItemRarityID.White;
+			Item.UseSound = SoundID.Item44;
+
+			Item.noMelee = true;
+			Item.DamageType = DamageClass.Summon;
+			Item.buffType = ModContent.BuffType<StarwoodSummonBuff>();
+			Item.shoot = ModContent.ProjectileType<StarwoodScepterSummonSplit>();
+		}
+
+		public override bool Shoot(Player player, EntitySource_ItemUse_WithAmmo source, Vector2 position, Vector2 velocity, int type, int damage, float knockback)
+		{
+			player.AddBuff(Item.buffType, 2);
+
+			Projectile proj = Projectile.NewProjectileDirect(source, Main.MouseWorld, Main.rand.NextVector2CircularEdge(5f, 5f), type, damage, knockback, player.whoAmI, ai2: 1f);
+			proj.originalDamage = Item.damage;
+
+			Projectile proj2 = Projectile.NewProjectileDirect(source, Main.MouseWorld, Main.rand.NextVector2CircularEdge(5f, 5f), type, damage, knockback, player.whoAmI, ai2: 0f);
+			proj2.originalDamage = Item.damage;
+
+			(proj.ModProjectile as StarwoodScepterSummonSplit).otherProj = proj2;
+			(proj2.ModProjectile as StarwoodScepterSummonSplit).otherProj = proj;
+
+			Helpers.DustHelper.DrawStar(Main.MouseWorld, ModContent.DustType<Dusts.GlowFastDecelerate>(), 5, 1, 1, 0.5f, 1, 0.5f, 0, -1, new Color(240, 200, 20));
+
+			return false;
+		}
+	}
+
+	public class StarwoodScepterSummonSplit : ModProjectile
+	{
+		public int lifetime;
+		public int empowermentTimer;
+		public int downtimeTimer;
+		public Projectile otherProj;
+		public Vector2 rotationalVelocity = Vector2.Zero;
+		public NPC MinionTarget
```

Comment:
The property with getters/setters should appear after autoproperties.

Additionally, linebreaks seperating fields from properties are neccisary, and the properties should likely be grouped and have linebreaks with reasonable relationships to each other